docs/world api.md

```diff
@@ -1,95 +1,95 @@
 # Archipelago API
 
-This document tries to explain some internals required to implement a game for
-Archipelago's generation and server. Once a seed is generated, a client or mod is 
-required to send and receive items between the game and server.
+This document tries to explain some aspects of the Archipelago World API used when implementing the generation logic of
+a game.
 
-Client implementation is out of scope of this document. Please refer to an
-existing game that provides a similar API to yours.
-Refer to the following documents as well:
-- [network protocol.md](https://github.com/ArchipelagoMW/Archipelago/blob/main/docs/network%20protocol.md)
-- [adding games.md](https://github.com/ArchipelagoMW/Archipelago/blob/main/docs/adding%20games.md)
+Client implementation is out of scope of this document. Please refer to an existing game that provides a similar API to
+yours, and the following documents:
 
-Archipelago will be abbreviated as "AP" from now on.
+* [network protocol.md](https://github.com/ArchipelagoMW/Archipelago/blob/main/docs/network%20protocol.md)
+* [adding games.md](https://github.com/ArchipelagoMW/Archipelago/blob/main/docs/adding%20games.md)
 
+Archipelago will be abbreviated as "AP" from now on.
 
 ## Language
 
 AP worlds are written in python3.
-Clients that connect to the server to sync items can be in any language that
-allows using WebSockets.
-
+Clients that connect to the server to sync items can be in any language that allows using WebSockets.
 
 ## Coding style
 
-AP follows [style.md](https://github.com/ArchipelagoMW/Archipelago/blob/main/docs/style.md).
-When in doubt use an IDE with coding style linter, for example PyCharm Community Edition.
-
+AP follows a [style guide](https://github.com/ArchipelagoMW/Archipelago/blob/main/docs/style.md).
+When in doubt, use an IDE with coding style linter, for example PyCharm Community Edition.
```

Comment:
The "a" should at least be here, less sure about "coding style" vs "code-style", but I think the latter is correct
```suggestion
When in doubt, use an IDE with a code-style linter, for example PyCharm Community Edition.
```